src/radical/entk/appman/appmanager.py

```diff
@@ -281,9 +300,10 @@ def run(self):
                     raise EnTKError("RabbitMQ setup failed")
```

Comment:
You never use `setup` again.  So, I suggest to either do:
```
self._mqs_setup = self._setup_mqs()
if not self._mqs_setup:
  raise...
```

or:

```
if not self._setup_mqs():
  raise...
```

But I would actually suggest to move that whole logic into the `self._setup_mqs()` method itself:

```
def _setup_mqs(self):
  if self.__setup_mqs_ok:
    return  # nothing to do
  
  try:
    do a
    do b
    ...
    self._setup_mqs_ok = True
  except ...:
    raise('RabbitMQ setup failed')
```

and then the code here simplifies to 

```
    self._setup_mqs()
```

Error handling should be localized to where the error can happen, not where the method is being called.

Reply:
Fair point. Will do :+1: